torch/onnx/utils.py

```diff
@@ -1132,9 +1132,7 @@ def _run_symbolic_function(g, block, n, inputs, env, operator_export_type=Operat
                         torch._C._jit_pass_onnx_node_shape_type_inference(new_node, _params_dict, opset_version)
                     return new_op_outputs
             else:
-                symbolic_name = "prim_" + op_name
-                domain = ""
-                symbolic_fn = _find_symbolic_in_registry(domain, symbolic_name, opset_version,
+                symbolic_fn = _find_symbolic_in_registry("prim", op_name, opset_version,
                                                          operator_export_type)
                 if symbolic_fn is None:
```

Comment:
Do you know why the symbolic function for PythonOp gets passed the Node `n` while other symbolic functions do not?
Even if we don't know the reason, please mention that in the documentation.

Reply:
@wschin There is a use case in ortmodule that need access more information for the node itself.

Reply:
In ORT, the function pointer to `autograd.Function` is retrieved via `name`. In the current Pytorch IR implementation, that `name` is not fully qualified. If two autograd.Fcuntsion's with the same exist, ORT doesn't know which to invoke. If we can access `ConcretePythonOp`, we can inspect the underlying Python object to get `qualname`.

Reply:
@jiafatom can you please add a comment to the code explaining this?

It's sort of ugly that's it's inconsistent, so maybe also file a work item and reference it here.

Reply:
Filed https://msdata.visualstudio.com/Vienna/_workitems/edit/1408006